test/setup_test_dbs.sh

```diff
@@ -1,11 +1,18 @@
 #!/bin/bash
 
-PG_DB_CONTAINER_NAME="pgtestdb"  # Also used in package.json to stop the container after the test
+# source the .testenv file to use its variables
+. test/.env
```

Comment:
Might be redundant to include both but I believe if it's running on a windows machine `. test/.env` would have to be replaced by `source test/.env`

Reply:
I don't believe we'll be developing or deploying on Windows machines anytime soon, but your point is well-taken that if that changes, we'll need to update several scripts to be cross-platform compatible.